src/main/java/mate/academy/onlinebookstore01/dto/book/CreateBookRequestDto.java

```diff
@@ -26,8 +26,8 @@ public class CreateBookRequestDto {
     private String author;
     @NotNull(message = ISBN_EXCEPTION)
     @Pattern(regexp
-            = "^(?:ISBN(?:-1[03])?:? )?(?=[0-9X]{10}$|(?=(?:[0-9]+[- ]){3})\n"
-            + "[- 0-9X]{13}$|97[89][0-9]{10}$|(?=(?:[0-9]+[- ]){4})[- 0-9]{17}$)\n"
+            = "^(?:ISBN(?:-1[03])?:? )?(?=[0-9X]{10}$|(?=(?:[0-9]+[- ]){3})"

```

Comment:
Try to avoid adding unrelated changes to PRs

Reply:
still exists, you can fix pattern issue with isbn in another pr.